apps/application-system/form/src/auth.ts

```diff
@@ -35,13 +35,13 @@ if (userMocked) {
       NationalRegistryScope.individuals,
       EndorsementsScope.main,
       ApiScope.internal,
+      ApiScope.internalProcuring,
       ApiScope.meDetails,
       ApiScope.fishingLicense,
       MunicipalitiesFinancialAidScope.read,
       MunicipalitiesFinancialAidScope.write,
       MunicipalitiesFinancialAidScope.applicant,
       ApiScope.assets,
-      ApiScope.noDebtCertificate,
```

Comment:
Is this scope being used anywhere else? If not, can it be deleted?

Reply:
It is not being used elsewhere, plan is to delete from IDS in the future, but keep it there until after this changes has gone into effect. Will make reverting these changes easier if needed.

Also I guess it will ensure tokens already released before will work until they are refreshed for the new scope as we still allow this scope until we delete it completely.